playground/kwj1270/commerce/display/src/main/resources/application.yml

```
@@ -3,4 +3,29 @@ server:
+      enabled: true
+  endpoints:
+    enabled-by-default: false
+    web:
+      base-path: /prefer
+      exposure:
+        exclude:
+        include: prometheus
+    jmx:
+      exposure:
+        exclude:
+        include: prometheus
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)